tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "baseUrl": ".",
+    "outDir": "./dist",
```

Comment:
can this break typings (as you explain in Discord)? or is this starter-kit not using them?

Reply:
This one hasn't been released to npm, so no impact yet, and I have to fix it anyway

Reply:
Checked typings aren't impacted, they ignore tsconfig (probably wrongly so)

Reply:
👍